src/theories/selftest.sml

```diff
@@ -0,0 +1,9 @@
+open HolKernel Parse boolTheory boolLib pairTheory
+
+open HolKernel Parse boolLib bossLib;
+open bir_bool_expSyntax;
+open bir_exp_substitutionsSyntax;
+open bir_extra_expsSyntax;
+open bir_interval_expSyntax;
```

Comment:
because this script does not work the other way around and actually the syntax files are have the theory files as dependency. if the theory files are not loaded the defined constants do not exist - because they are actually not defined.

i think opening theories without doing anything with them is redundant. they have been successfully compiled from the scripts and loading them without using them would be a check of the hol theory generation code.

Reply:
I didn't see this that way, them this makes perfect sense!